pkg/userdata/rhel/provider.go

```diff
@@ -315,6 +311,11 @@ write_files:
     [Service]
     EnvironmentFile=-/etc/environment
 
+rh_subscription:
```

Comment:
Would it be worthy to make this section optional?

Reply:
After some more thinking I prefer to leave this aside for the moment, as we do not provide other means of registering.